documentation/meta/contribution/good_first_and_help_wanted_issues.md

```diff
@@ -0,0 +1,194 @@
+- Are limited to single programming language or programming paradigm (i.e,
+  frontend JS, CSS, and markup)
+- Occupy a single slice of the Openverse stack
+- Touch a single file or as few files as necessary; ideally these files are
+  colocated in the repository
+- Are based on the `main` branch and not sub-features of a larger, multi-PR
+  changeset
+
+Other qualities of appropriate "good first" issues are that they:
+
+- Are not 'high priority', 'critical priority', or otherwise time sensitive
+- Are appropriate for those new to open source or Git workflows in-general
+- Can be, approximately, completed within two hours, one of which may be
+  necessary for local development environment setup
+
+## Writing good first issue descriptions
+
+Good first issue descriptions should provide as much context as possible. These
+issues should generally be self-contained; all the documentation necessary to
+complete the issue should be included with or linked to in the issue
+description. The aim is to make the issue as approachable as possible while
+setting clear expectations. This up-front effort is to the benefit of
+contributors and maintainers. Ideally, it decreases the amount of time
+maintainers spend reviewing PRs that have not met the minimum requirements
+(e.g., no unit tests, linting failures, and so on). For contributors, it helps
+prevent them from feeling discouraged after completing the "core functionality"
+of a PR to then have to revisit the PR and add significant changes.
+
+Descriptions should:
+
+- Clearly articulate the expected outcome of resolving the issue. Include any of
+  the following, when relevant:
+  - Changes (modifications or additions) to tests
+  - Link to instructions for updating Playwright snapshots if the change might
+    require it
+  - Changes (modifications or additions) to documentation
+- Link to the relevant files and/or lines of code which need changing
+- Reference any past PRs which will help the contributor, for example:
+  - A PR with a similar change
+  - A PR which originally implemented the feature being modified
+- Relevant Openverse domain knowledge, either links to succinct documentation or
+  links and a summary of the important aspects when documentation is detailed
+  - Ideally "good first issue"s require essentially no Openverse-specific domain
+    knowledge, but if they at all do (for example, to explain why a non-obvious
+    solution is requested) then it must be included
+
+### Additional boilerplate for good first issue templates
+
+In addition to our standard issue template, good first issues should _also_
+contain the following block of requirements:
+
+```md
+## Good first issue checks
+
+- Have I filled out the PR template correctly?
+  - Did I include testing instructions?
+- Does my PR pass linting? (Test either via precommit or by running `just lint`
+  manually)
+- Do all the tests still pass?
+  - If JavaScript changes, run `pnpm -r run test`
```

Comment:
`just frontend/run test` should work here (for consistency with the other commands)

Reply:
The problem with `just frontend/run test` is that it _only_ runs frontend tests. `pnpm -r run test` runs `test` for all workspaces, which will include the eslint plugin, which we've had contributors open PRs for. The test suites run parallel to each other, so it doesn't add any extra time, and prevents any chance of someone getting confused about _where_ the changes they made are relevant. Especially important if we have anything in packages in the future that is depended on by the frontend and not explicitly tested in the frontend suite (which makes sense, we wouldn't test dependencies normally). Using `pnpm -r run test` here (and ideally everywhere) prevents it from being overlooked in the future.

Reply:
Thank you for explaining!